### ISSUE TITLE
Reusable Template Not Found

### ISSUE THREAD

**Issue by MidnightJava** [prometheus/alertmanager#2270]
**What did you do?**
I defined a receiver and a template location in my alert manager config as shown below.
I created a simple reusable template and saved it in the template file I point to in the config. I created this simple template after my intended template was not found, in case there was something wrong with it. But even this simple template is not found.
{{ define "custom.email" }}Test{{ end }}

I run the alertmanager:latest docker image, and I verify that /etc/alertmanager/templates/custom-email-template.tmpl contains the template text as expected.

**What did you expect to see?**
I expected alertmanager to find the template I defined and start successfully.

**What did you see instead? Under which circumstances?**
When I run the docker container, alertmanager fails immediately, with error: template "custom.email" not defined

AFAICT I am following precisely the example here: https://prometheus.io/blog/2016/03/03/custom-alertmanager-templates/ but I must be missing something. I tried many variations, and I cannot get alertmanager to recognize my defined template no matter what I do.

**Environment**

* System information:

	Linux 3.10.0-10629.1.el7.x86_64 x86_64

* Alertmanager version:

	alertmanager, version 0.20.0 (branch: HEAD, revision: f74be0400a6243d10bb53812d6fa408ad71ff32d)
build user: root@00c3106655f8
biold date: 20191211-14:13:14
go version: gol.13.5

* Alertmanager configuration file:

global:
  smtp_smarthost: ''my.host:25'
  smtp_hello: 'my.system'
  smtp_from: 'my.adderss'

route:
  receiver: test-email-receiver
  group_by: [label1, label2]

receivers:
- name: 'test-email-receiver'
  email_configs:
  - to: 'someone@example.org'
    text : '{{ template "custom.email" . }}'

templates:
  - '/etc/alertmanager/templates/custom-email-template.tmpl'

**Comment by roidelapluie** [comment]
Thanks for your report. It looks as if this is actually a question about usage and not development.

It makes more sense to ask questions like this on the [prometheus-users mailing list](https://groups.google.com/forum/#!forum/prometheus-users) rather than in a GitHub issue. On the mailing list, more people are available to potentially respond to your question, and the whole community can benefit from the answers provided.

In your case, please also use `'{{ template "custom-email-template.tmpl" . }}'` and verify that your templates are mounted in a volume in the container.

**Comment by MidnightJava** [comment]
Hi, thanks for your response. I posted here because I think it's broken. I wanted to acknowledge that maybe I'm missing something about how to use it, but the instructions are very simple, and it doesn't seem to work. I did send a message to the prometheus-users mailing list, and so far there has been no reply. My apologies if I am indeed missing something, but I think this is a bug, not a usage question. And I do confirm that the directory with my custom templates is mounted in the container. I exec into the running container and find the custom template exactly where I specified it to be in the config file.
